services/simcore/docker-compose.yml

```diff
@@ -138,6 +158,13 @@ services:
       placement:
         constraints:
           - node.labels.simcore==true
+      resources:
+        reservations:
+          cpus: "0.1"
+          memory: "64M"
+        limits:
+          cpus: "0.1"
+          memory: "128M"
```

Comment:
same here. how was this defined? maybe we could try running the new puppet 100 times in parallel and see what happens? cause if the limit is hit, then the static-webserer will be killed.

Reply:
As discussed in whereby